java/runtime/src/main/java/org/ray/runtime/raylet/MockRayletClient.java

```diff
@@ -20,11 +23,13 @@
   private final Map<UniqueId, Map<UniqueId, TaskSpec>> waitTasks = new ConcurrentHashMap<>();
   private final MockObjectStore store;
   private final RayDevRuntime runtime;
+  private final ExecutorService exec ;
 
   public MockRayletClient(RayDevRuntime runtime, MockObjectStore store) {
     this.runtime = runtime;
     this.store = store;
     store.registerScheduler(this);
+    exec  = Executors.newFixedThreadPool(5);
```

Comment:
sometimes, we want to debug and process task one by one. In such case, it would be better to set the thread pool size = 1. So make it configurable here.

Reply:
we can name the config item something like `ray.dev-runtime.execution-parallelism`